source/nextflow.rst

```diff
@@ -33,7 +33,7 @@ Parameters
    * - VPCId
      - Choose VPC Id in the drop-down list.
    * - WorkerNodeSubnetId
-     - Subnet you want your Batch Worker Node to launch in. We recommend public subnets.
+     - Choose Subnet Id inthe drop-down list.
```

Comment:
I think we should just leave the earlier line which was more descriptive. If you still want to change it, please put a space between "in" and "the".